README.md

```diff
@@ -20,6 +20,7 @@ Dual-licensed under [MIT](LICENSE-MIT) or [Apache 2.0](LICENSE-APACHE)
 
 - [Installation](#install)
 - [Getting Started](#getting-started)
+- [GitHub Action](docs/github-action.md)
```

Comment:
Should we instead reference this down in the "Integrations" section?

Reply:
Sure thing!

Reply:
Actually looking at the section I think I disagree - this is talking more about command line usage of typos. GitHub Actions I think is going to be the only interest for some folks, and if we just remove it there and hope they read down further, they won't see it. So I think we should keep it at the top level as a separate thing so that someone coming to the repo _looking_ for it explicitly sees it quickly.

Reply:
Its talking about that, because that is all there is for now.  I'd put native integrations, like Github Actions, before instuctions for custom integrations (seeing as this technically would be building on the stuff documented in the custom integrations)

Reply:
okay, I'll try to extend the table of contents to  have GitHub actions under integrations.